### DISCUSSION TITLE
Hello All , I would like show the image from the File path example from c:/foldername/x.png

### DISCUSSION THREAD

**Discussion by srirayavarapu** [microsoft/react-native-windows#9869] [General]
Hello All , I would like show the image from the File path example from c:/foldername/x.png. When I have given file path, we are getting cross mark but its not loading the image, any one has any idea?
Code which I was working 

 <Image  
         source={require('C:/Users/username/Pictures/66.png')} 
        
        
         
          style={{
            width:220,
            height:220,
            borderWidth:2,
            borderColor:'#d35647',
            resizeMode:'stretch',
            margin:8
          }}
        />

**Comment by acoates-ms** [comment]
You should use relative URLs from the app project.  I'm not sure that the asset import will work correctly specifying a full path to something outside of the app project.

Or if you really want it to load the file at that location, you could use source='file://c:/Users/username/Pictures/66.png', but that would require that the image is already on the machine that installs the app -- and generally UWP apps will not have access to that file location.

**Reply by srirayavarapu** [reply]
my image location is PC (knownfolders - like pictures ), but even giving like "file://c:/Users/username/Pictures/66.png", image does not load, I have given app to broaderfilesystem access as well. So something wrong with Image control
